src/mockApartments.js

```diff
@@ -9,7 +9,7 @@ let mockApartments = [
     price: "1800",
     bedrooms: 5,
     bathrooms: 3.0,
-    pets: "yes!",
+    about: "This is a trial run for what an about me will look",
```

Comment:
Just remember if you decide to change your apartment data you will need to run a migration and update all your seed data, validations, and tests.